ga4gh/datamodel/__init__.py

```diff
@@ -27,6 +27,44 @@ def _cleanupHtslibsMess(indexDir):
         shutil.rmtree(indexDir)
 
 
+class CompoundId(object):
+    """
+    Base class for compoundIds
+    """
+    seperator = ':'
+
+    def __init__(self, compoundIdStr):
+        try:
+            splits = compoundIdStr.split(self.seperator)
+        except AttributeError:
+            raise exceptions.BadIdentifierException(
+                compoundIdStr, self._getParseErrorMessage())
+        if len(splits) != len(self.fields):
```

Comment:
Where does ``self.fields`` come from? If it's an attribute, shouldn't we be passing it in to the constructor?